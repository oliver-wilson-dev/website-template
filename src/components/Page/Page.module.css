@import "../../styles/globalVars.css";
@import "../../styles/colours.css";

.page {
  margin-top: $navigation-height;
  background-color: $background-grey;
  flex-grow: 1;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: flex-start;
  font-size: calc(10px + 2vmin);
  color: black;
}
