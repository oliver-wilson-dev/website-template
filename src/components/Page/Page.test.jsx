import React from 'react';
import { shallow } from 'enzyme';
import Page from './Page';

const render = () => shallow(<Page><h1>hello world</h1></Page>);

it('renders correctly', () => {
  expect(render()).toMatchSnapshot();
});
