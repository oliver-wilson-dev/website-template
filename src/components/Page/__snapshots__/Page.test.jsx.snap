// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="page"
>
  <h1>
    hello world
  </h1>
</div>
`;
